core/node/libp2p/rcmgr.go

```diff
@@ -0,0 +1,37 @@
+package libp2p
+
+import (
+	"context"
+	"fmt"
+	"github.com/ipfs/go-ipfs/repo"
+	"github.com/libp2p/go-libp2p"
+	"github.com/libp2p/go-libp2p-core/network"
+	rcmgr "github.com/libp2p/go-libp2p-resource-manager"
+	"go.uber.org/fx"
+)
+
+func ResourceManager() func(fx.Lifecycle, repo.Repo) (network.ResourceManager, Libp2pOpts, error) {
+	return func(lc fx.Lifecycle, repo repo.Repo) (network.ResourceManager, Libp2pOpts, error) {
+		var limiter *rcmgr.BasicLimiter
+		var opts Libp2pOpts
+
+		// FIXME(BLOCKING): Decide how is the `limit.json` file path going to be consumed,
+		//  either by default in the repo root or through the `go-ipfs-config`.
+		limiter = rcmgr.NewDefaultLimiter()
```

Comment:
@schomatis It should read the file from the directory from where `ipfs daemon` is run. There shouldn't be any config option. Could you please implement this?